SKILL.tsv

```diff
@@ -1780,7 +1780,7 @@ SKILL_20150317_001779	Increase knockdown power
 SKILL_20150317_001780	Time is stopped.
 SKILL_20150317_001781	{memo X}Max. strength is reduced, attack will increase
 SKILL_20150317_001782	Increased attacking damage.
-SKILL_20150317_001783	Resisting to knock down and Knockback.
+SKILL_20150317_001783	Resistant to knock down and Knockback.
```

Comment:
Resistant to knock down and Knockback. -> Resistant to knock down and knock back.